Arena.toml

```diff
@@ -1870,11 +1850,6 @@ document = "stjudecloud/workflows:/tools/bwa.wdl"
 message = "bwa.wdl:263:16: note[ShellCheck]: Double quote to prevent globbing and word splitting."
 permalink = "https://github.com/stjudecloud/workflows/blob/8377299f9d171af316f06570eedcd12e39600d72/tools/bwa.wdl/#L263"
 
-[[diagnostics]]
-document = "stjudecloud/workflows:/tools/bwa.wdl"
-message = "bwa.wdl:268:13: note[ShellCheck]: Double quote to prevent globbing and word splitting."
-permalink = "https://github.com/stjudecloud/workflows/blob/8377299f9d171af316f06570eedcd12e39600d72/tools/bwa.wdl/#L268"
-
```

Comment:
This concatenates a variable to a string, but the string has embedded quotes that surround the variable value.